### ISSUE TITLE
insertChild's atIndex ignored on iOS when using FlexboxLayout

### ISSUE THREAD

**Issue by ludwiktrammer** [NativeScript/NativeScript#3481]
### Tell us about the problem
On iOS when using the flexbox layout a child added using the `insertChild` method is always added at the end, not matter what the `atIndex` parameter is. The problem doesn't occur with other or with Android.

### Which platform(s) does your issue occur on?
iOS

### Please provide the following version numbers that your issue occurs with:
- CLI: (run `tns --version` to fetch it)
2.4.2

- Cross-platform modules: (check the 'version' attribute in the
`node_modules/tns-core-modules/package.json` file in your project)
2.4.4

- Runtime(s): (look for the `"tns-android"` and `"tns-ios"` properties in the
`package.json` file of your project)
2.4.0

### Please tell us how to recreate the issue in as much detail as possible.
I use this code to test the issue:

XML:
```
<FlexboxLayout id="layout">
    <Label text="World" />
</FlexboxLayout>
```

TS:
```
let layout = <Layout>component.getViewById("layout");
let label = new Label();
label.text = "Hello";
layout.insertChild(label, 0);
```

On Android the result is "Hello World" (correct). On iOS the result is "World Hello" (incorrect).

**Comment by tsonevn** [comment]
Hi @ludwiktrammer,
Thank you for reporting this problem.

I reviewed your case on my side and confirm that this is a real problem for iOS, when adding new element using `insertChild` method and specify the position.

I am also attaching the sample code, which I used for testing.

main-page.xml
```
<Page xmlns="http://schemas.nativescript.org/tns.xsd" navigatingTo="navigatingTo">
    <FlexboxLayout loaded="fbloaded" id="layout">
        <Label text="World" />
    </FlexboxLayout>
</Page>
```
main-page.ts
```
export function fbloaded(args){
    let layout:FlexboxLayout= <FlexboxLayout>args.object;
    let label = new Label();
    label.text = "Hello";
    layout.insertChild(label, 0);
}
```
Regards,
@tsonevn

**Comment by ludwiktrammer** [comment]
It seems that NativeScript 2.5 fixed the issue :)

**Comment by tsonevn** [comment]
Hi @ludwiktrammer,

Indeed this issue has been fixed with 2.5.

Regards,
@tsonevn
